src/styles/components/form-elements/styles.less

```diff
@@ -35,9 +41,20 @@
     position: absolute;
   }
 
-  .form-remove {
+  .form-group-container-action-button {
```

Comment:
That's true, the element with this class is not the actual button. And I could see it containing more than one button.

What do you think about `.form-group-container-action-button-group`? It's a bit wordy but it does make sense to me...